scalafix/rules/src/main/scala/fix/JsonParse.scala

```diff
@@ -0,0 +1,40 @@
+package fix
```

Comment:
I guess we should add the migration to Scala Steward when releasing so our users profit more directly, right?

Reply:
Yes But I need this PR to be merged, this is the plan afterwards.